pages/common/install.md

```diff
@@ -0,0 +1,24 @@
+# install
+
+> Copy files and set attributes.
+> Provide a standardized way of manipulating a file/directory's ownership and permissions while copying file or creating directory.
```

Comment:
Are you sure that's not the intent/main use of the command? Here's what I get from [this man page](https://linux.die.net/man/1/install):

> copies files (often just compiled) into destination locations you choose. If you want to download and install a ready-to-use package on a GNU/Linux system, you should instead be using a package manager like yum(1) or apt-get(1).

And from [this one](http://ss64.com/bash/install.html):

> It is typically used in Makefiles to copy programs into their destination directories.

We could either embrace the current meaning of "install" as in "install a program" and explain that this is the typical usage given to this command, as I've previously suggested, or we could revert to the strict etymological meaning of "install" as "place something in a given place", but in that case we'd still have to make the connection explicitly (mention "install" as a synonym for whatever expression we use). I think the first option is easier to accomplish in 1-2 lines of a main description.